shop/src/main/java/ro/msg/learning/shop/utils/mapper/ProductMapper.java

```diff
@@ -0,0 +1,48 @@
+package ro.msg.learning.shop.utils.mapper;
+
+import ro.msg.learning.shop.dto.ProductDto;
+import ro.msg.learning.shop.dto.ProductToSaveDto;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.model.ProductCategory;
+import ro.msg.learning.shop.model.Supplier;
+
+public class ProductMapper {
+    public static ProductDto ProductToDto(Product product) {
+        if (product == null)
+            return null;
+        return new ProductDto(product.getId(), product.getName(), product.getDescription(),
```

Comment:
Same here, pass the product as the only parameter and use its attributes in the constructor

Reply:
yes :) i deleted this one as it was unused but in regards to the above mentioned requested changes i complied with them as the mapper looks much easier now and it's plain to understand it.

Reply:
Implemented the highly necessary constructor.  Seeing this, I may think that I could've used @Builder but I wasn't so aware of its existence at first. But this method is for sure a life saver.